pymfit/masking.py

```diff
@@ -319,6 +331,17 @@ def make_mask(image, thresh=1.5, backsize=110, backffrac=0.5,
     obj, seg, bkg, img = detect_sources(
         img, thresh, backsize, backffrac,
         None, True, kern_sig, **sep_extract_kws)
+    
+    sbuf = np.zeros_like(image, dtype='uint8')
```

Comment:
Only run if switch is on. Change `use_old_filter` to name that describes algorithm.